app/src/main/res/layout/list_item.xml

```diff
@@ -1,26 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
In #19 this file is delete and a new list item file is create. Please check it out.